packages/caravan-bitcoin/src/multisig.ts

```diff
@@ -115,10 +115,16 @@ export function generateMultisigFromRaw(addressType, multisig) {
       return payments.p2sh({ redeem: multisig });
     case P2SH_P2WSH:
       return payments.p2sh({
-        redeem: payments.p2wsh({ redeem: multisig }),
+        redeem: payments.p2wsh({
+          redeem: multisig,
+          witness: [multisig.output],
+        }),
       });
     case P2WSH:
-      return payments.p2wsh({ redeem: multisig });
+      return payments.p2wsh({
+        redeem: multisig,
+        witness: [multisig.output],
```

Comment:
This resulted in witness script being added to UTXO, but perhaps there's a more appropriate package in which to do this than in `caravan-bitcoin`?

Reply:
Interesting. Yeah would be nice to keep this fix for the newer code/packages. This means that either we're still using a legacy caravan/bitcoin function for the psbt generation (or its constituent parts), OR one of the newer functions is still using some old code that still has the bug. 

Tracking down where this is used, `generateMultisigFromRaw` is called from `generateMultisigFromPublicKeys` and `generateMultisigFromHex`. The latter is used in a newer version of `getUnchainedInputsFromPSBT` which is used in `translatePSBT` which I only see being used in the legacy ledger and trezor interactions and `generateMultisigFromPublicKeys` doesn't seem to be really used anywhere relevant either... 

it's also used in `multisigScriptSig` which is used in `signedMultisigTransaction` which... isn't used anywhere. 

So I'm curious why this change was even necessary to have an impact.